src/schema/metadata/BackgroundSuppressionNumberPulses.yaml

```diff
@@ -0,0 +1,8 @@
+---
+key_name: BackgroundSuppressionNumberPulses
+description: |
+  The number of background suppression pulses used.
+  Note that this excludes any effect of background suppression pulses applied
+  before the labeling.
+type: integer
+minimum: 0
```

Comment:
Is `minimum` equivalent to `inclusiveMinimum`?

Reply:
> Is minimum equivalent to inclusiveMinimum?

conceptually yes, but according to the JSON schema spec, there is no dedicated keyword for `inclusiveMinimum`: https://json-schema.org/understanding-json-schema/reference/numeric.html?highlight=minimum#range



        x ≥ minimum
        x > exclusiveMinimum
        x ≤ maximum
        x < exclusiveMaximum

Reply:
related: https://github.com/bids-standard/bids-validator/pull/1261

Reply:
Awesome! I'll replace `Minimum` with `minimum` and `inclusiveMaximum` with `maximum`. Thanks!